src/indexation/pht.cpp

```diff
@@ -25,39 +25,30 @@ namespace dht {
 namespace indexation {
 
 /**
- * Output the blob into string and readable way
+ * Get bit string representation of a blob for a specified length.
  *
- * @param bl   : Blob to print
+ * @param bl   A Blob
+ * @param len  The number of bits to consider.
  *
- * @return string that represent the blob into a readable way
+ * @return string that represent the blob
  */
-static std::string blobToString(const Blob &bl) {
+std::string Prefix::firstBitsToString(const Blob &bl, size_t len) const {
```

Comment:
If you choose to make it a non-static method, you should remove the second argument and checks, and use size_ directly (otherwise shall remain static).